deploy/charts/rook-ceph/values.yaml

```diff
@@ -373,6 +373,7 @@ csi:
   #  image: registry.k8s.io/sig-storage/csi-attacher:v3.5.0
   #resizer:
   #  image: registry.k8s.io/sig-storage/csi-resizer:v1.5.0
+  #imagePullPolicy: IfNotPresent
```

Comment:
```suggestion
  #  imagePullPolicy: IfNotPresent
```
let's follow syntax

Reply:
But it's a property of csi not resizer so I just followed the same syntax as for resizer,registrar,..